Formula/calceph.rb

```diff
@@ -3,7 +3,7 @@ class Calceph < Formula
   homepage "https://www.imcce.fr/inpop/calceph"
   url "https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-3.4.6.tar.gz"
   sha256 "fc2c0d899c32ddee2d484159c6fda5ae1ee7d8beae517029b776e5250b5fda27"
-  license "GPL-2.0-or-later"
+  license "CECILL-2.1"
```

Comment:
Thanks for the fix, @gastineau!

```suggestion
  license any_of: ["CECILL-2.1", "CECILL-B", "CECILL-C"]
```

`CECILL-B` and `CECILL-C` aren't OSI approved so I'm not 100% sure we want to include them. I'd like another maintainer's opinion here.

Reply:
Are they allowed by debian? Cause that's our benchmark

Reply:
There are two CECILL-licensed packages in Debian main:
* simpleburn: [CECILL-2.0](https://metadata.ftp-master.debian.org/changelogs//main/s/simpleburn/simpleburn_1.8.0-1_copyright)
* scotch: [CECILL-C](https://metadata.ftp-master.debian.org/changelogs//main/s/scotch/scotch_6.0.6-2_copyright)

Also, the only substantive difference I've found between CECILL-2.0 and CECILL-2.1 is that the former is explicitly compatible only with GPL, while the latter adds AGPL and EUPL. I guess CECILL-2.1 should also be OK.

However, there was a claim that CECILL-B [may not pass Debian's "desert island test"](https://lists.debian.org/debian-legal/2018/12/msg00080.html), so it's probably best to leave that one out.